conf/nginx.conf

```diff
@@ -28,5 +28,5 @@ location {LOCATION} {
 }
 
 location @wallbag2 {
-  rewrite ^ {PATH}/app.php$is_args$args;
+  rewrite ^ {PATH}/app.php?is_args$args;
```

Comment:
Ok, I'll close your PR so. Thank you anyway for trying to fix the problem with bookmarklet!